script.js

```diff
@@ -1,12 +1,304 @@
+
+  let theArticle = document.createElement("article"); // Creates an article element.
+  theArticle.id = "article"; //Assigned the id of article to the article.
+  theArticle.setAttribute("class", "artContainer"); // Assigned the class of artContainer to the Article element.
+  theContainer.appendChild(theArticle); // Appended the Article to the div of our placeholder.
+
+  let theTitleTagName = document.createElement("h3"); //Created the a h3 element to hold the title.
+  theArticle.appendChild(theTitleTagName); // Appended the h3 element to the article Container.
+  theTitleTagName.textContent = `${show.name}`; /*-${titleMaker(
+    element.season,
+    element.number
+  )}`;*/ // Set the the text content of the h3 element the object name property called the title maker maker function
+  theTitleTagName.id = "title"; // Set the id of h3 element to 'title'.
+
+  let theImageTagName = document.createElement("img"); // created an image element.
+  theArticle.appendChild(theImageTagName); // Appended the image tag to the article parents.
+  theImageTagName.id = "image"; //Set an id of 'image to it'
+
+  var noImage = "";
+  show.image == null ? noImage : (noImage = show.image.medium);
+  theImageTagName.src = noImage;
+  //Assigned the medium image object property to img element.
+  let theParagraph = document.createElement("p"); // Created Paragraph element.
+
+  theParagraph.textContent = show.summary.replace(/(<([^>]+)>)/gi, ""); //set the paragraph text content to the summary object property and remove the preset p tag using replace method
+  theParagraph.id = "summary"; //Set the id of summary to the paragraph element.
+  theArticle.appendChild(theParagraph); // Appended the paragraph to the article
 }
 
-function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+/* This function renders All The content of the page */
+function makePageForEpisodes(showId) {
+  rootElem.textContent = ""; // Set the of the root to blank as my the root content is the parent of my elements
+
+  if (showId != 0) {
+    /*This will calls the Async function gives it show id parameter*/
+    fetchEpisodesAsync(showId).then((episodesData) => {
+      // receives a episodesData
+
+      episodeDropList(episodesData, showId); //  calls episode drop list function Passes episodesData and showId
+      episodesData.forEach((e) => {
+        // loops through the episodes data
+
+        //calls the function that creates html for episodes
+        setEpisodesHTML(e);
+      });
+    });
+  } else {
+    makePageForShows(allShows); // displays all the shows.
+  }
+}
+
+/*This function will create the basic structure of one episode Title image and description that takes element as an argument*/
+
+/*4th creates episode content */
+function setEpisodesHTML(element) {
+  //console.log(element);
```

Comment:
your code is nice and interesting but if I were you I removed my console logs after my functions start working.